driver/examples/snapshots/pvcfrmsnap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvcfrmsnap1
```

Comment:
Hi Smita, Can we use some good names instead of mysnapshot, snap1,mysnapcontent etc